hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1623,23 +1623,22 @@
             the HMAC done and FIFO empty interrupts as a part of this test.
             '''
       milestone: V2
-      tests: []
+      tests: ["chip_sw_hmac_enc_irq", "chip_sw_hmac_enc_idle"]
     }
     {
       name: chip_sw_hmac_idle
       desc: '''Verify the HMAC clk idle signal to clkmgr.
 
-            - Write the HMAC clk hint to 1 within clkmgr to indicate HMAC clk is ready to be gated.
-              Verify that the HMAC clk hint status within clkmgr reads 0 (HMAC is idle).
-            - Initiate an HMAC operation with a known key, plain text and digest.
-              Verify that the HMAC clk hint status within clkmgr now reads 1 (HMAC is not idle),
-              before the HMAC operation is complete.
+            - Program the HMAC clk hint within clkmgr to indicate HMAC clk is ready to be gated.
+              Verify that the HMAC clk enabled status within clkmgr reads disabled (HMAC is idle).
```

Comment:
You will be reading the CLK_HINTS_STATUS CSR so "hint status" may be clearer. Same below.

Reply:
Done.